File: Amazon/ServerSide/Amazon.Solution/Amazon.Core/Entities/Wishlist.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Amazon.Core.Entities
{
    public class Wishlist
    {
        public Wishlist()
        {
        }
        public Wishlist(string id)
        {
            Id = id;
        }

        [Required]
        public string Id { get; set; }
        public List<WishlistItem> Items { get; set; } = new List<WishlistItem>();
    }
}

File: Amazon/ClientSide/Amazon.ClientSide/src/app/Models/wish-list.ts
import { WishListItem } from "./wish-list-item";
 
export class WishList 
{
    id: string
    items: WishListItem[]

    constructor(id:string, items:WishListItem[])
    {
        this.id = id;
        this.items = items
    }
}
